orangewidget/utils/itemmodels.py

```diff
@@ -161,7 +161,10 @@ def _argsortData(self, data: numpy.ndarray, order):
         Return indices of sorted data. May be reimplemented to handle
         sorting in a certain way, e.g. to sort NaN values last.
         """
-        indices = numpy.argsort(data, kind="mergesort")
+        if data.ndim == 1:
+            indices = numpy.argsort(data, kind="mergesort")
+        else:
+            indices = numpy.lexsort(data.T[::-1])
```

Comment:
I know this is fine, because I did the same thing in feature statistics, but I don't see where this would be used.

Reply:
`OWPredictions` uses it to sort by discrete probabilities. Both changes in this commit come from what I need there. I could derive a new class from this one (or make a new class from scratch), but I thought these changes are general enough and could also be useful elsewhere.